packages/recomponents/src/components/r-button-group/r-button-group.scss

```diff
@@ -11,6 +11,8 @@
 .r-button-group .r-button {
     margin-left: -1px;
     margin-left: -0.1rem;
+    margin-right: -1px;
+    margin-right: -0.1rem;
```

Comment:
I think for cases like `-1px` it's better not to use rem at all to avoid possible issues with convertation from rem to px or weird root value